.circleci/config.yml

```
@@ -75,6 +75,17 @@ jobs:
       - run:
           name: Run RuboCop
           command: rubocop
+      - run:
+          name: Install eslint
+          command: yarn install
+      - run:
+          name: Run eslint
+          command: yarn lint --quiet --format junit --output-file ~/reports/eslint.xml
```

```suggestion
          command: yarn lint --quiet --format junit --output-file ~/reports/eslint/result.xml
```
If you do this rename, circleci will attribute the output to the correct tool.